2주차/화_12947/12947_고혁진.java

```diff
@@ -0,0 +1,25 @@
+class Solution {
+    public boolean solution(int x) {
+        boolean answer = true;
+        int y = x;
+        int z = x;
+        int n = 1;
+        int sum = 0;
+        while (x > 9) {
+            x /= 10;
+            n += 1;
+        }
+        for (int i = n-1; i >= 0; i--) {
+            if(Math.pow(10,i)==0)
+                continue;
+            sum += y / Math.pow(10,i);
+            y %= Math.pow(10,i);
+        }
+        if (z % sum == 0) {
+            answer = true;
+        } else {
+            answer = false;
+        }
+        return answer;
+    }
```

Comment:
```suggestion
        return z % sum == 0;
    }
```
해당 if문을 요렇게 축약해도 좋을거 같아요~

Reply:
오 굿아이디어!